code/modules/hydroponics/grown/nymph.dm

```diff
@@ -23,4 +23,15 @@
 	new /mob/living/simple_animal/diona(get_turf(user))
 	to_chat(user, "<span class='notice'>You crack open [src] letting the nymph out.</span>")
 	user.drop_item()
-	qdel(src)
\ No newline at end of file
+	qdel(src)
+
+// The seed granted to a diona at round start.
+/obj/item/seeds/nymph/diona_innate
+	desc = "A single seed produced by a small diona gestalt. It can typically only grow one nymph."
+	name = "diona nymph seed"
+
+// The Diona's innate pod will not yield more than one nymph at harvest and perennial growth cannot be added (see plant_genes.dm).
+// A seed extractor can possibly make more, but if the player is in a position to do that they can just get the nymph seeds from the vendor anyway.
+/obj/item/seeds/nymph/diona_innate/getYield()
```

Comment:
Seeing as how the parent `getYield()` is a proc, I believe you need to add in "proc" before the `getYield()`.
Like this: `/obj/item/seeds/nymph/diona_innate/proc/getYield()`

Reply:
getYield() is a proc in /obj/item/seeds which is altered in this type. Adding /proc/ would throw a duplicate definition error; I know. I've tried.